### DISCUSSION TITLE
Question about submit result

### DISCUSSION THREAD

**Discussion by zyf-zone** [PaddlePaddle/PaddleSpatial#48] [BaiduKDDCup2022] [answered]
![1652185740(1)](https://user-images.githubusercontent.com/39518571/167628340-cf3b40d8-d5b7-4a41-b412-b9b2b7512615.png)
I noticed this module, my question is will all team submissions show up here? If not, will there be a public list? Or do we only know the winner at the end? Thanks.

**Comment by zhoujingbo** [comment, marked as answer]
The public list and winners will show up.
